packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -1172,7 +1172,7 @@ class _LineItemState extends State<LineItem>
 
     final word = wordForHover(event.localPosition.dx, widget.lineContents);
 
-    if (word != '') {
+    if (word != '' && !isPrimitiveValueOrNull(word)) {
```

Comment:
can this be `word.isNotEmpty` instead?

Reply:
will follow up with word is not empty check here